clang/lib/DPCT/APIMapping/QueryAPIMapping.h

```diff
@@ -12,28 +12,22 @@
 #include <string>
 #include <unordered_map>
 
+#include "llvm/ADT/DenseMap.h"
```

Comment:
Let's follow the LLVM coding style, include the LLVM headers before system headers.
```
Main Module Header
Local/Private Headers
LLVM project/subproject headers (clang/..., lldb/..., llvm/..., etc)
System #includes
```